scripts-test/weather.py

```
@@ -0,0 +1,60 @@
+
+def get_weather(city):
+    base_url = f"http://wttr.in/{city}?format=j1"
+    headers = {
+        "User-Agent": "Mozilla/5.0"
+    }
+    response = requests.get(base_url, headers=headers)
+    response.raise_for_status()  # This will raise an HTTPError if the request failed
+    weather_data = response.json()
+    return weather_data
+
+def print_weather(weather_data):
```

Ruff (0.8.2)

Missing return type annotation for public function `print_weather`

Add return type annotation: `None`

(ANN201)